src/main/java/org/apache/commons/lang3/builder/ToStringStyle.java

```diff
@@ -467,6 +472,7 @@ protected void removeLastFieldSeparator(final StringBuffer buffer) {
      *  for summary info, <code>null</code> for style decides
      */
     public void append(final StringBuffer buffer, final String fieldName, final Object value, final Boolean fullDetail) {
+        if (omitNulls && value == null) return;
```

Comment:
With our current checkstyle checks, this will create an error (error in Checkstyle, not a build error).